tests/test_copy_novaseqx_flow_cell.py

```diff
@@ -0,0 +1,126 @@
+from pathlib import Path
+from typing import Set
+from cg.cli.demultiplex.copy_novaseqx_data import get_latest_analysis_directory
+
+from cg.cli.demultiplex.demux import (
+    copy_flow_cell_analysis_data,
+    is_ready_for_post_processing,
+)
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+def test_flow_cell_is_ready_for_post_processing(
+    novaseqx_flow_cell_dir: Path, demultiplexed_flow_cell_run_directory: Path
+):
+    # GIVEN a flow cell which is ready for post processing
+
+    # WHEN checking if the flow cell is ready for post processing
+    ready = is_ready_for_post_processing(
+        novaseqx_flow_cell_dir, demultiplexed_flow_cell_run_directory
+    )
+
+    # THEN the flow cell is ready
+    assert ready
+
+
+def test_is_not_ready_without_analysis(
```

Comment:
```suggestion
def test_flow_cell_is_ready_for_post_processing_without_analysis(
```
I think this links the tests, function and test  scenario more clearly.